src/Controllers/Setup/strings.php

```diff
@@ -67,8 +67,9 @@
         }
     }
     /* Make sure log directory exists as well */
-    if (!is_dir(dirname(Config::$log_file))) {
-        mkdir($dirname(Config::$log_file), 0755);
+    $log_dir = dirname(Config::$log_file);
+    if (!is_dir($log_dir) || !mkdir($log_dir, 0755)) {
+        die("Could not create log directory at '$log_dir'");
```

Comment:
I presume this was one of the changes mentioned? Shouldn't there be less quotes and more concat dot's here?

Reply:
That's silly, it should be &&